.github/workflows/build.yml

```diff
@@ -224,6 +226,26 @@ jobs:
         name: docker-image
         path: artifacts
 
+    - name: Generate sha256 checksums
+      if: github.event_name == 'release'
+      run: |
+        for filePath in $(find  artifacts/* -name '*.tar.gz' -or -name '*.zip')
+        do
+          fileName="$(basename -- $filePath)"
+          newSha=$(sha256sum "$filePath" | awk '{ print $1 }')
```

Comment:
Calculate the `sha256` checksum for the file and grab just the checksum (the output from `sha256sum` includes the file path as well)